### ISSUE TITLE
OAS 3 -> Swagger 2: `produces` is converted incorrectly when using `default` responses key.

### ISSUE THREAD

**Issue by hannesgeodan** [LucyBot-Inc/api-spec-converter#175]
In OAS3 we specified the expected responses of our api as follows:

```
paths:
  /foo:
    get:
      responses:
        '200':
          description: Foo
          content:
            application/hal+json:
              schema:
                $ref: '#/components/schemas/Foo'
        default:
          description: Unexpected problem
          content:
            application/problem+json:
              schema:
                $ref: '#/components/schemas/Problem'
      tags:
        - Foos
      operationId: getFoo
```

However, when we convert this (using 2.7.10) to Swagger 2 markup, we get this:

```
paths:
  /foo:
    get:
      produces:
        - application/problem+json
      parameters: []
      responses:
        '200':
          description: Foo
          schema:
            $ref: '#/definitions/Foo'
        default:
          description: Unexpected problem
          schema:
            $ref: '#/definitions/Problem'
      tags:
        - Foos
      operationId: getFoo
```

However, we expected to get this:

```
paths:
  /foo:
    get:
      produces:
        - application/hal+json
        - application/problem+json
      parameters: []
      responses:
        '200':
          description: Foo
          schema:
            $ref: '#/definitions/Foo'
        default:
          description: Unexpected problem
          schema:
            $ref: '#/definitions/Problem'
      tags:
        - Foos
      operationId: getFoo
```

So, the `produces` keyword is filled incorrectly (wrongly sorted and filtered).

**Comment by bobby-brennan** [comment]
Thanks for the report. This should be fixed in v2.7.11

**Comment by hannesgeodan** [comment]
@bobby-brennan  Thanks for your quick fix! I upgraded and tested it, and it works! Perfect.
